packages/next/lib/helpers/install.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
This file, along with the other `helper/` files in the PR, were ported over directly from 'package/create-next-app`.

If there is a preferred way to re-use the same logic without duplicating these files, let me know 🙏